packages/fluent-ui/src/ColorWidget/ColorWidget.tsx

```diff
@@ -39,21 +36,18 @@ const allowedProps = [
 ];
 
 const ColorWidget = ({
-  id,
   schema,
   options,
   value,
   required,
-  disabled,
-  readonly,
+  // disabled,
+  // readonly,
   label,
   onChange,
-  onBlur,
-  onFocus,
+  // onBlur,
+  // onFocus,
```

Comment:
Delete this maybe since it is not used... or perhaps you meant to uncomment?

Reply:
```suggestion
```